### DISCUSSION TITLE
Scaling of atom types

### DISCUSSION THREAD

**Discussion by chemsh** [deepmodeling/deepmd-kit#4004] [General]
Dear DeePMD-kit Gurus,

we are developers of ChemShell--a world leading software package for multiscale simulations. We are very keen on interfacing DeePMD-kit to Py-ChemShell, whereas we have several general enquiries:

- It's said (we can't yet confirm) that DeePMD-kit needs typically 170-250 KB data for training a system of 4-5 atom types, for example in the paper Chem. Sci., 2020, 11, 2335. Is this (still) true in DeePMD-kit v3? How does the data size scale wrt. the number of atom types? For example, what would a typical data size be for a solvated protein system?

- Do you have a plan (or already an ongoing project) to introduce the GNN technologies (e.g., E(3)-equivariant) to DeePMD-kit?

Thank you very much.

Kind regards,

You

**Comment by wanghan-iapcm** [comment]
- In the mentioned paper, they do NOT have the type embedding feature introduced in dpa-1, and the data requirement for multi-component systems would be heavy.  I strongly recommend you to use dpa-1/2 that has the  type embedding feature for multi-component systems. As an example, we have trained dpa-1/2 with ~200K data for a >30 component system, and the models present excellent accuracy. 
- GNN and E(3)-equivariant are different concept. dpa-2 is GNN. The introduction of E(3)-equivariant would not lead to obvious improvement to dpa-2

**Reply by 1292211370** [reply]
Hi, I used the  example/water/dpa2/input_torch_small.json to train DPA2 model. I noticed that it does not include pair-wise attention in the Repformer in a smallest model. In this case, can DPA2 still be considered a GNN?

**Reply by wanghan-iapcm** [reply]
yes, it is still a GNN model.
